gapis/service/service.proto

```diff
@@ -397,6 +397,9 @@ message ErrDataUnavailable {
   stringtable.Msg reason = 1;
   // If true, then making the same request at a later time may result in data.
   bool transient = 2;
+  // If true, draw call specific data was requested for a command range not
+  // containing any draw calls.
+  bool not_a_draw_call = 3;
```

Comment:
> OK, let's make ErrDataUnavailable be a "soft" error in the UI and if we find cases where the server returns it, instead of a "hard" error, we can fix it. Sounds good to me.

Awesome, thanks.

> So, should the UI just assume that ErrDataUnavailable means it's because the user didn't select a draw call? Probably not...

Nope, that seems like a bad idea.

> I did add the ERR_NOT_A_DRAW_CALL message, and we could add other messages for any new reason the mesh may not be available at the given command/tree node

Yup, I'm happy with that as it fits with the current model.

> We could possibly get away with just displaying that message and make it general enough

That was the design. Sorry that wasn't clear. Please make a note of things like this if they're not obvious as it's clear we need to document this.

> The UI could look at the message identifier and display its own message instead of the server's message if it's ERR_NOT_A_DRAW_CALL...

That was the thought process I was having with the proto enum of all the message types, but its nasty and doesn't support additional information. After an hours worth of thinking, I believe we should steer clear of any sort of special casing of the stringtable tags. If we _really_ want to indicate to the client some bespoke error reason, I think we need to consider more `ErrXXX` proto messages.